.github/workflows/main.yaml

```diff
@@ -73,7 +73,7 @@ jobs:
     - uses: actions/checkout@v2
 
     - name: setup
-      run: make prepare-host
+      run: makim host.prepare
```

Comment:
this step will not work correctly because in order to makim you need it installed:

https://github.com/thegraphnetwork/EpiGraphHub/pull/181/checks#step:3:1

so you can install it using something similar to this step: https://github.com/thegraphnetwork/epigraphhub_py/blob/main/.github/workflows/build.yml#L36

you probably need to change some values there

Reply:
I don't understand very well, could you explain a little more

Reply:
sure thing.
if you check the steps, there is no step that install the dependencies, and you are trying to use makim that is not installed. "make" by default is installed on github actions for linux.

so in order to access makim before this setup step, you need to installed it first. you can do it using that step I mentioned before that install the conda stack into the CI.

let me know if you need more details please. thanks

Reply:
The detail is that the line you pointed out to us is already in the main.yaml file or is this
https://github.com/thegraphnetwork/epigraphhub_py/blob/main/.github/workflows/build.yml#L46

Reply:
oh nice .. I didn't see that .. so it would be much easier .. just move it above, just after the checkout step .. I think that will do the trick